### DISCUSSION TITLE
Update not working

### DISCUSSION THREAD

**Discussion by tobehn** [Budibase/budibase#2717] [Help] [answered]
Hi BB-Team,
since v 0.9.136 i cannot update on my selfhosted setup anymore by pressing the check for Updates button.
Is this meant to be as i see watchtower saying there is `no new images found for /bbworker`.

From your Relaeses i can see you have published 0.9.141 already.

Do i miss anything?

**Comment by mjashanks** [comment, marked as answer]
This will fetch the `latest` tag from DockerHub, which is still on `0.9.136`.

**Reply by tobehn** [reply]
i thought it might automatically be the latest non-alpha release in your releases.

**Reply by shogunpurple** [reply]
@tobehn We slightly changed the way releases work. We used to automatically roll out everything to self hosters when we merged to the main branch, but now we push tags insteaad so that we can test changes in our cloud environment before pushing them out to everyone. You can update your `docker-compose.yaml` to use these tags if you really want to use the newest updates, but I'd advise waiting until we release them properly for stability purposes.

**Reply by tobehn** [reply]
@shogunpurple i was curious about #2643 being solved in one of the latest PRs.

But that's fine i don't want to waste your time with my individual request - telling me how to use the bleeding edge.
I'll be patient 🤓
